packages/hub/main.js

```diff
@@ -53,6 +50,33 @@ async function wireItUp(projectDir, encryptionKeys, dataSources, opts = {}) {
   return container;
 }
 
+async function startIndexing(environment, container) {
+  // some datasources are dependent upon a sync at boot for index of pristine system
+  await container.lookup('hub:indexers').update();
+
+  if (environment !== 'production') {
+    let ephemeralStorage = await container.lookup(`plugin-services:${require.resolve('@cardstack/ephemeral/service')}`);
```

Comment:
I think this will throw if you don't happen to have ephemeral installed?

Reply:
good point